package.json

```diff
@@ -3,7 +3,9 @@
   "description": "PBC web site",
   "version": "1.0.0",
   "authors": [
-    "Eric Soderberg"
+    "Eric Soderberg",
+    "Jim Thornton",
+    "Google LLC"
```

Comment:
Just curious why we need "Google LLC" as an author?

Reply:
This is one of the very few rules for me to submit patches to an open source project like this: to list as author if the project has an authors listing.